riptide-nakadi/src/main/java/org/zalando/riptide/nakadi/Batch.java

```diff
@@ -0,0 +1,32 @@
+package org.zalando.riptide.nakadi;
+
+/*
+ * ⁣​
+ * Riptide: Nakadi
+ * ⁣⁣
+ * Copyright (C) 2015 - 2016 Zalando SE
+ * ⁣⁣
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * ​⁣
+ */
+
+import java.util.List;
+
+import lombok.Value;
+
+@Value
+final class Batch {
+
+    Cursor cursor;
```

Comment:
Cursor should be of type JsonNode because we don't care about the actual content. Maybe a package private `@JsonValue JsonNode getValue()` to hide that from clients.